fbpic/fields/fields.py

```diff
@@ -496,12 +590,16 @@ def erase(self, fieldtype ) :
         ---------
         fieldtype : string
             A string which represents the kind of field to be erased
-            (either 'E', 'B', 'J', 'rho')
+            (either 'E', 'B', 'J', 'rho' or 'A')
```

Comment:
For now, since we do not need the `erase` method, I would recommend reverting it to its previous version (without A)